debian/clearwater-monit.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
Is this file only used in the RPM build, not the deb build?
If so, can we have a comment stating what it's for